OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -314,7 +314,7 @@ public enum SubCell : byte { Invalid = byte.MaxValue, Any = byte.MaxValue - 1, F
 
 	public interface IPositionableInfo : IOccupySpaceInfo
 	{
-		bool CanEnterCell(World world, Actor self, CPos cell, Actor ignoreActor = null, bool checkTransientActors = true);
+		bool CanEnterCell(World world, Actor self, CPos cell, Actor ignoreActor = null, bool checkTransientActors = true, SubCell subCell = SubCell.FullCell);
```

Comment:
Can we please define `subCell` after `cell` in the argument list for consistency with other cell/subcell accepting methods? I have just checked that this is a straightforward swap, with only `Production` needing `ignoreActor` but not `subCell`.